Update Cloud of Darkness.md (#49)
naseffxiv/nase.gg: ffxivResources/chaotic/Dawntrail Chaotic Alliance Raids/Cloud of Darkness.md

## Rewards

The raid's rewards include two new mounts, Clouddark gear (iLvl 730 Left-side Only Gear), and a new unique hairstyle.
The Clouddark gear has some options with different substats from either tome or savage gear. NOTE: There is NO weekly lockout for this fight.

> 1 clear gets you 2 Demimateria I and 1 Demimateria II. You will also receive for the following loot to roll on; Guarenteed 2 Loot Cofferes for Left side gear, a chance for 1 of the 2 mounts, and a chance for the the hairstyle. This is rolled only between the 8 members in your alliance. For any additional folks that it was their first clear, you get 2 additional Demimateria II per person present in the run who it was their first time clearing.

Reviewer: Changed some spelling stuff and reorganized some information -- feel free to disregard this comment though! 

```suggestion
1 clear gets you 2 Demimateria I and 1 Demimateria II. You get 2 additional Demimateria II per person present in the run who are clearing the fight for the first time.

You will also receive the following loot to roll on; guaranteed 2 Loot Coffers for Left side gear, a chance for 1 of the 2 mounts, and a chance for the the hairstyle. This is rolled only between the 8 members in your alliance. 
```
